src/OrbitGl/GlCanvas.cpp

```diff
@@ -27,6 +27,7 @@ float GlCanvas::kZValueTrack = 0.01f;
 float GlCanvas::kZValueIncompleteDataOverlayPicking = 0.02f;
 float GlCanvas::kZValueEventBar = 0.03f;
 float GlCanvas::kZValueBox = 0.05f;
+float GlCanvas::kZValueBoxBorder = 0.07f;
```

Comment:
This shouldn't be needed. Lines are drawn after boxes, so you should be able to just use the `kZValueBox` layer. I assume you introduced this because you render the box border on top of the box?